source/producers/ESSStream/KafkaPlugin.h

```diff
@@ -73,6 +73,14 @@ class KafkaConfigPlugin : public DAQuiri::Plugin
     uint16_t kafka_decomission_wait_{5000};
 
     static std::string random_string( size_t length );
+
+    /// \brief load daquiri settings from environment variables
+    void getEnvironmentSettings();
+
+    // Environment overrides
+    struct  EnvironmentSettings_t {
```

Comment:
As it currently is, I see no good reason to have a struct containing only one member.